pkg/controllers/nodeclaim/lifecycle/liveness.go

```diff
@@ -55,7 +55,7 @@ func (l *Liveness) Reconcile(ctx context.Context, nodeClaim *v1beta1.NodeClaim)
 	if err := l.kubeClient.Delete(ctx, nodeClaim); err != nil {
 		return reconcile.Result{}, client.IgnoreNotFound(err)
 	}
-	logging.FromContext(ctx).With("ttl", registrationTTL).Debugf("terminating due to registration ttl")
+	log.FromContext(ctx).WithValues("ttl", registrationTTL).V(1).Info("terminating due to registration ttl")
```

Comment:
Can we make this v(0)? Seems it shouldn't have been debug to begin with imo. Its very important information to know if a nodeclaim has been terminated for liveness.

Reply:
Yes, we can.